.template/.rubocop.yml

```diff
@@ -0,0 +1,53 @@
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.7
+  Exclude:
+    - 'vendor/**/*'
+
+Lint/NonDeterministicRequireOrder:
+  Exclude:
+    - '../spec/rails_helper.rb'
+
+Metrics/MethodLength:
+  Exclude:
+    - '../template.rb'
+    - '**/template.rb'
+    - 'hooks/**/*'
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Enabled: false
+
+Metrics/CyclomaticComplexity:
+  Exclude:
+    - '../template.rb'
+    - '**/template.rb'
+
+Metrics/PerceivedComplexity:
+  Exclude:
+    - '../template.rb'
+    - '**/template.rb'
+
+Naming/FileName:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+# TODO: Enable this rule later
+Style/GlobalVars:
+  Enabled: false
+
+# TODO: Enable this rule later
```

Comment:
I'll enable these rules later in another PR, so this PR is not too big. 🙏

Reply:
@rosle , here are two of my favorites cops:
- [Style/GuardClause](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause)
- [Style/OneLineConditional](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/OneLineConditional)

Is this something we could incorporate, what do you think? Thank you!

Reply:
Noted. I'll open more PRs to fix these cops and I'll revisit those rules 👍 ✨

Reply:
@tyrro I revisited these 2 cops, seems we already have it enabled 👍 🚀